{{ cookiecutter.repo_name }}/{{ cookiecutter.pkg_name }}/config/settings/dev.py

```diff
@@ -20,7 +20,8 @@ def INTERNAL_IPS(self):
         return (self.get_addr(),)
 
     # devserver must be ahead of django.contrib.staticfiles
-    INSTALLED_APPS = ('devserver',) + Common.INSTALLED_APPS + ('debug_toolbar',)
+    #INSTALLED_APPS = ('devserver',) + Common.INSTALLED_APPS + ('debug_toolbar',)
+    INSTALLED_APPS = Common.INSTALLED_APPS + ('debug_toolbar',)
```

Comment:
I thought we would like to use django-devserver in the future. But atm latest version is not working with Django 1.8. We could use some of these fork. What do you think?